Downgrade warning:

```
/home/vsts/work/1/s/src/Tools/Microsoft.dotnet-openapi/test/dotnet-microsoft.openapi.Tests.csproj : error NU1605: Detected package downgrade: NuGet.Frameworks from 5.11.0 to 5.10.0-rc.7240. Reference the package directly from the project to select a different version.  [/home/vsts/work/1/s/.dotnet/sdk/7.0.100-alpha.1.21555.1/NuGet.targets]
/home/vsts/work/1/s/src/Tools/Microsoft.dotnet-openapi/test/dotnet-microsoft.openapi.Tests.csproj : error NU1605:  Microsoft.DotNet.Open.Api.Tools.Tests -> Microsoft.NET.Test.Sdk 17.1.0-preview-20211109-03 -> Microsoft.TestPlatform.TestHost 17.1.0-preview-20211109-03 -> Microsoft.TestPlatform.ObjectModel 17.1.0-preview-20211109-03 -> NuGet.Frameworks (>= 5.11.0)  [/home/vsts/work/1/s/.dotnet/sdk/7.0.100-alpha.1.21555.1/NuGet.targets]
/home/vsts/work/1/s/src/Tools/Microsoft.dotnet-openapi/test/dotnet-microsoft.openapi.Tests.csproj : error NU1605:  Microsoft.DotNet.Open.Api.Tools.Tests -> NuGet.Frameworks (>= 5.10.0-rc.7240) [/home/vsts/work/1/s/.dotnet/sdk/7.0.100-alpha.1.21555.1/NuGet.targets]
    0 Warning(s)
    1 Error(s)
```

--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -212,9 +212,9 @@
     <MicrosoftWebAdministrationVersion>11.1.0</MicrosoftWebAdministrationVersion>
     <MicrosoftWebXdtVersion>1.4.0</MicrosoftWebXdtVersion>
     <SystemIdentityModelTokensJwtVersion>6.10.0</SystemIdentityModelTokensJwtVersion>
-    <NuGetPackagingVersion>5.10.0-rc.7240</NuGetPackagingVersion>
-    <NuGetVersioningVersion>5.10.0-rc.7240</NuGetVersioningVersion>
-    <NuGetFrameworksVersion>5.10.0-rc.7240</NuGetFrameworksVersion>
+    <NuGetPackagingVersion>5.11.0</NuGetPackagingVersion>
+    <NuGetVersioningVersion>5.11.0</NuGetVersioningVersion>
+    <NuGetFrameworksVersion>5.11.0</NuGetFrameworksVersion>
     <SystemComponentModelAnnotationsVersion>5.0.0</SystemComponentModelAnnotationsVersion>
     <SystemNetExperimentalMsQuicVersion>5.0.0-alpha.20560.6</SystemNetExperimentalMsQuicVersion>
     <SystemSecurityPrincipalWindowsVersion>5.0.0</SystemSecurityPrincipalWindowsVersion>

